pirania-app/files/www/portal/js/voucher.js

```diff
@@ -35,98 +42,104 @@ function authVoucher () {
-      errorElem.innerHTML = int[lang].invalid
-      show(errorElem)
-    } else if (res.error) {
-      console.log(res.error)
-      errorElem.innerHTML = res.error
+      voucherElem.value = ''
+    })
+    .catch(err => {
+      console.log('Erro no Ubus', err)
+      errorElem.innerHTML = err
       show(errorElem)
-
       ubusError = true
-    }
-    voucherElem.value = ''
-  })
-  .catch((err) => {
-    console.log('Erro no Ubus', err)
-    errorElem.innerHTML = err
-    show(errorElem)
-    ubusError = true
-  })
-
+    })
 }
 
-function getIp() {
-  window.RTCPeerConnection = window.RTCPeerConnection || window.mozRTCPeerConnection || window.webkitRTCPeerConnection;//compatibility for Firefox and chrome
-  var pc = new RTCPeerConnection({iceServers:[]}), noop = function(){};      
-  pc.createDataChannel('');//create a bogus data channel
-  pc.createOffer(pc.setLocalDescription.bind(pc), noop);// create offer and set local description
-  pc.onicecandidate = function(ice) {
+function getIp () {
+  window.RTCPeerConnection =
+    window.RTCPeerConnection ||
+    window.mozRTCPeerConnection ||
+    window.webkitRTCPeerConnection // compatibility for Firefox and chrome
+  var pc = new RTCPeerConnection({ iceServers: [] })
+
+  var noop = function () {}
+  pc.createDataChannel('') // create a bogus data channel
+  pc.createOffer(pc.setLocalDescription.bind(pc), noop) // create offer and set local description
+  pc.onicecandidate = function (ice) {
     if (ice && ice.candidate && ice.candidate.candidate) {
-      var myIP = /([0-9]{1,3}(\.[0-9]{1,3}){3}|[a-f0-9]{1,4}(:[a-f0-9]{1,4}){7})/.exec(ice.candidate.candidate)[1];
-      pc.onicecandidate = noop;
+      var myIP = /([0-9]{1,3}(\.[0-9]{1,3}){3}|[a-f0-9]{1,4}(:[a-f0-9]{1,4}){7})/.exec(
+        ice.candidate.candidate
+      )[1]
+      pc.onicecandidate = noop
       if (myIP) userIp = myIP
       else {
         fetch('http://thisnode.info/cgi-bin/client_ip')
-        .then(i => {
-          console.log(i)
-          return JSON.parse(i) 
-        })
-        .then((res) => {
-          console.log('Ubus res: ', res)
-          if (res && res.result[1]) {
-            userIp = res.result[1]
-          } else if (res.error) {
-            userIp = res.error
+          .then(i => {
+            console.log(i)
+            return JSON.parse(i)
+          })
+          .then(res => {
+            console.log('Ubus res: ', res)
+            if (res && res.result[1]) {
+              userIp = res.result[1]
+            } else if (res.error) {
+              userIp = res.error
+              ubusError = true
+            }
+          })
+          .catch(err => {
+            console.log('Erro no Ubus', err)
             ubusError = true
-          }
-        })
-        .catch((err) => {
-          console.log('Erro no Ubus', err)
-          ubusError = true
-        })
+          })
       }
     }
   }
 }
 
-function getValidClients() {
+function getValidClients () {
```

Comment:
RECOMMENDATION IF Commits about format changes are separated, it shortens the review length, making it easier for reviewers

Reply:
Auto-save linter lints everything, sorry.

Reply:
This happened to me all the time, now I usually open the file, save (this does the autolint), commit the style, and then continue with the changes.

Reply:
:+1: Good idea marcos